### DISCUSSION TITLE
a lazer play on an unranked map decreased my stable accuracy

### DISCUSSION THREAD

**Discussion by averyunderstars** [ppy/osu#22201] [Q&A] [answered]
so recently i set [a score](https://lazer.ppy.sh/scores/2175091932) on [this map](https://lazer.ppy.sh/beatmapsets/1565666#osu/3197058)

unfortunately i don't have a screenshot, but after passing the map and viewing more details, i saw my overall accuracy go down 10 percent, from around 97.7% to circa 86%. currently, my accuracy is at 86.79%, [which you can view on the normal site](https://osu.ppy.sh/users/20746542); [as well as the lazer version](https://lazer.ppy.sh/users/20746542). is this intended?

**Comment by Theighlin** [comment, marked as answer]
It's intended that lazer changes your profile accuracy, though i don't know the math of why it dropped so much. As far as i know, setting a play on stable should reset it.

**Reply by averyunderstars** [reply]
Yup, it changed: 
![screenshot233](https://user-images.githubusercontent.com/87026073/212537574-67b526e6-a9c6-4afe-943d-85e1bd66c7f6.jpg)

I'm pretty sure I set stable scores beforehand, but maybe they just weren't my top scores on the maps
